### PR TITLE
Fixes #12958 - calling update_capabilities only from host

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: app/assets/javascripts/host_edit.js [hunk 1]
```diff
@@ -263,15 +263,6 @@ function set_inherited_value(hostgroup_elem) {
   })
 }
 
-function organization_changed(element) {
```
Comment (orrabin) [inline app/assets/javascripts/host_edit.js:266]: This seems to be unused.

Comment (domcleal) [inline app/assets/javascripts/host_edit.js:266]: It's used by the org/loc selects, see host_taxonomy_select in the hosts_helper.

**Comment by orrabin** [comment]
@domcleal thanks, I added back the methods for org/loc changed.

**Comment by domcleal** [comment]
Merged as 18f4ac7, thanks @orrabin.

**Pull request closed without merging**